README.md

```diff
@@ -36,6 +36,70 @@ GitHub Action to run Renovate self-hosted.
 | <a href="https://renovatebot.com"><img alt="Renovate enabled" src="https://img.shields.io/badge/renovate-enabled-brightgreen.svg?style=flat-square"></a>                                                                | Dependencies | Renovate             |
 | <a href="https://github.com/renovatebot/github-action/actions"><img alt="GitHub workflow status" src="https://img.shields.io/github/actions/workflow/status/renovatebot/github-action/build.yml?style=flat-square"></a> | Build        | GitHub Actions       |
 
+## Quick Start
+
+### `Step 1: Renovate Explained`
+Renovate is a nifty tool for automatically updating your projects dependencies so you don't have to manually! Lets go through how to setup a GitHub Action for Renovate.
+
+### `Step 2: Generate Personal Access Token (PAT)`
```

Comment:
DO NOT use PAT!! This is a wrong approach and it's a security and maintenability nightmare. 


You do not need any tokens in your repository since thre are company-wide token issues and managed by Developer Experience team and they are available in all repositories